packages/central-server/src/apiV2/dashboardMailingListEntries/GETDashboardMailingListEntries.js

```diff
@@ -58,6 +59,11 @@ export class GETDashboardMailingListEntries extends GETHandler {
       ],
       dbOptions.multiJoin,
     );
-    return { dbConditions, dbOptions };
+
+    const filterDbConditions = {
```

Comment:
Including criteria back into conditions, as was not included in getDashboardsDBFilter